apps/block/google_news_result.ts

```diff
@@ -67,7 +67,7 @@ export class GoogleNewsResult extends SearchResultToBlock {
     );
     this.valid = true;
     this._canRetry = true;
-    this.url = href;
+    this.url = href ? href : "";
```

Comment:
This won't work. Cannot block when href is null/empty/undefined, so `this.valid = false` and `this._canRetry = false`. There is a code in L50 for when titleElement === null, so please do the same thing there.